### ISSUE TITLE
deepfm 中的

### ISSUE THREAD

**Issue by Lu1352** [ZHAN007/RecSystem#1]
deepFM中 linear_logit = Linear(use_bias=True)(linear_inputs)，
layers中 LInear 它的输入这样，是不是有问题？
  class Linear(Layer):
    def call(self, inputs):
        # [batch_size * feat_num * emb_size, batch_size * feat_num]
        sparsez_emb, dense_value = inputs
        linear_logit = tf.reduce_sum(sparsez_emb, axis=1) + self.dense(dense_value)
        if self.use_bias:
            linear_logit += self.bias
            
        return linear_logit

**Comment by ZHANSHANJUN** [comment]
哈哈哈是的，这个我没有完全统一接口，layers我后面更新了，deepfm没有更新，我最近会更新。。
